composer-symfony5.4.json

```diff
@@ -61,6 +62,7 @@
         "symfony/twig-bundle": "5.4.*",
         "symfony/uid": "5.4.*",
         "symfony/validator": "5.4.*",
+        "symfony/web-profiler-bundle": "5.4.*",
```

Comment:
when in 'prod' (Docker staging) environment, PHPUnit was complaining that these two bundles weren't available